apollo-ios/Sources/ApolloAPI/Selection.swift

```diff
@@ -6,7 +6,7 @@ public enum Selection {
   /// An inline fragment with a child selection set nested in a parent selection set.
   case inlineFragment(any InlineFragment.Type)
   /// A fragment spread or inline fragment marked with the `@defer` directive.
-  case deferred(if: Condition? = nil, any Deferrable.Type, label: String?)
+  case deferred(if: Condition? = nil, any Deferrable.Type, label: String)
```

Comment:
A label on the defer directive is required by the JS frontend validation, this enforces that in the generated models. It's also required for fulfilled fragments to work.